applications/tari_dan_app_utilities/src/base_layer_scanner.rs

```diff
@@ -290,11 +290,10 @@ impl BaseLayerScanner {
 
             for output in utxos.outputs {
                 let output_hash = output.hash();
-                let sidechain_feature = output.features.sidechain_feature.as_ref().ok_or_else(|| {
-                    BaseLayerScannerError::InvalidSideChainUtxoResponse(
-                        "Validator node registration output must have a sidechain features".to_string(),
-                    )
-                })?;
+                let Some(sidechain_feature) = output.features.sidechain_feature.as_ref() else {
+                    warn!(target: LOG_TARGET, "Validator node registration output must have sidechain features");
```

Comment:
Which sidechain outputs is the base node returning that have this problem?

Reply:
I have a chain that has old burnt outputs without the sidechain feature set. This prevents the VN from scanning. 

There may be a use case for a burn without the Confidential Output sidechain feature (not sure) otherwise we should add a base layer validation.

Reply:
Yeah I think let's leave this as is for now, but I'll get an issue created for the base layer validation